crates/db_schema/src/utils.rs

```diff
@@ -287,37 +288,35 @@ pub fn is_email_regex(test: &str) -> bool {
   EMAIL_REGEX.is_match(test)
 }
 
-pub fn diesel_option_overwrite(opt: Option<String>) -> Option<Option<String>> {
-  match opt {
+/// Takes an API text input, and converts it to an optional diesel DB update.
+pub fn diesel_string_update(opt: &Option<String>) -> Option<Option<String>> {
```

Comment:
Should probably be taking an `Option<&String>`, maybe even `Option<&str>`.  [See here](https://stackoverflow.com/questions/71026147/reference-to-an-option-versus-option-of-a-reference).

Reply:
I tried that at first, but it forces you to add a lot of deref calls. This one uses the least amount of cloning and mutating.

Reply:
Does calling `as_ref` on the param at the site of the function call work for you? It should let you turn, e.g., an `&Option<String>` to `Option<&String>`. [Docs for reference.](https://doc.rust-lang.org/std/option/enum.Option.html#method.as_ref)

Reply:
Should at least prevent us from needing to clone and pass owned objects around as much, which I think is a win.